src/app/screens/Settings.tsx

```diff
@@ -56,6 +57,21 @@ function Settings() {
           )}
         </Setting>
 
+        <Setting
+          title="Legacy LNURL Auth"
```

Comment:
I'd call this maybe "Legacy messaging signing for LNDhub and LNbits" 
because this changes message signing (and not only/specifically LNURL-Auth).  

Ideally we also only show this IF the user has an lndhub or lnbits account? there is an `getAccounts` action that returns all accounts (config is encrypted). We could check in that array for a connector === 'lndhub' || connector === 'lnbits' ? what do you think?

Reply:
Ok this would be a good idea, but I think it would be even more useful if we somehow get the time when an account was added, and don't show this setting if that is after a certain time?

Reply:
nice idea. but we don't have the time or what are you're ideas? 

I would have left it in for maybe ~2months and then we remove the deprecated option. I think that's OK at this stage.

Reply:
That's also possible, but then we need to make sure that users now that they need to empty their old accounts.

Reply:
![Schermafbeelding 2022-03-30 om 14 43 11](https://user-images.githubusercontent.com/33457577/160836949-ccb43eb0-39c5-4db3-96c6-9048085ebe61.png)

I would just leave it at this then to be honest. People will only visit this page when they have this problem with their legacy accounts anyway.

Reply:
Is this text ok, @bumi ?

Reply:
to base it on time we would need to store more information, don't we?